.github/workflows/auto-approve.yaml

```
@@ -0,0 +1,20 @@
+---
+name: Renovate Bot dependency updates auto-merge
+
+permissions:
+  pull-requests: write
+  contents: write
+
+on: pull_request_target
```

We can no longer use pull_request_target trigger (related thread: https://kongstrong.slack.com/archives/C0DT09NJE/p1735924399767949)

```suggestion
on:
  pull_request:
    types: ['opened']
```